src/main/steps/applicant/task-list/get.ts

```diff
@@ -1,9 +1,250 @@
+//import { CosApiClient } from '../../../app/case/CosApiClient';
+import autobind from 'autobind-decorator';
+import { Response } from 'express';
+
+import { AppRequest } from '../../../app/controller/AppRequest';
 import { GetController } from '../../../app/controller/GetController';
 
 import { generateContent } from './content';
+//import { CosApiClient } from '../../../app/case/CosApiClient';
+//import { RESPONDENT_ORDERS_FROM_THE_COURT } from '../../../steps/urls';
 
+@autobind
 export default class ApplicantTaskListGetController extends GetController {
   constructor() {
     super(__dirname + '/template', generateContent);
   }
+  public async load(req: AppRequest, res: Response): Promise<void> {
+    // const citizenUser = req.session.user;
+    // const caseId = req.session.userCase.id;
+    // const client = new CosApiClient(citizenUser.accessToken, 'https://return-url');
+    // const hearings = await client.getAllHearingsForCitizenCase(citizenUser, caseId);
+    req.session.userCase.hearingCollection = [
```

Comment:
Please refer PRl-3914 PR for code reference to load hearing data

Reply:
done this.